src/py21cmfast/drivers/coeval.py

```diff
@@ -756,11 +733,7 @@ def _setup_ics_and_pfs_for_scrolling(
 
     # We can go ahead and purge some of the stuff in the initial_conditions, but only if
     # it is cached -- otherwise we could be losing information.
-    with contextlib.suppress(OSError):
-        initial_conditions.prepare_for_perturb(
-            astro_options=inputs.astro_options, force=always_purge
-        )
-
+    initial_conditions.prepare_for_perturb(force=True)
```

Comment:
Can you explain why we are now always forcing the purge?

Reply:
Good catch, I will change this to depend on the `CacheConfig`